advanced/helm/databases/mysql-ei/values.yaml

```diff
@@ -0,0 +1,758 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+mysql:
+  imageTag: "5.7"
+  mysqlRootPassword: root
+  mysqlUser: wso2carbon
+  mysqlPassword: wso2carbon
+  fullnameOverride: "wso2ei-rdbms-service-mysql"
+  configurationFiles:
+    mysql.cnf: |-
+      [mysqld]
+      max_connections = 10000
+  initializationFiles:
+    init.sql: |-
+      DROP DATABASE IF EXISTS WSO2EI_USER_DB;
+      DROP DATABASE IF EXISTS WSO2EI_INTEGRATOR_CONFIG_GOV_DB;
+      DROP DATABASE IF EXISTS WSO2_CLUSTER_DB;
+      DROP DATABASE IF EXISTS EI_ANALYTICS;
+      DROP DATABASE IF EXISTS WSO2_CARBON_DB;
+      DROP DATABASE IF EXISTS WSO2_PERSISTENCE_DB;
+      DROP DATABASE IF EXISTS WSO2EI_MB_DB;
+
+      CREATE DATABASE WSO2EI_USER_DB;
+      CREATE DATABASE WSO2EI_INTEGRATOR_CONFIG_GOV_DB;
+      CREATE DATABASE WSO2_CLUSTER_DB;
+      CREATE DATABASE EI_ANALYTICS;
+      CREATE DATABASE WSO2_CARBON_DB;
```

Comment:
@Savidude IMO, this database is not required as this is not used anywhere. Please correct me if I am wrong.

Reply:
Fixed https://github.com/wso2/kubernetes-ei/pull/180/commits/4eab0894505a022e409f632e44196a919575ceb1